cli/012-cli-admin-usage-ux.md

```diff
@@ -0,0 +1,65 @@
+- Start Date: 2021-08-22
+- RFC PR:
+- Issue:
+
+### Simplify admin commands usage
+
+Problem: As of today admin commands are way too complex to use as they require passing DB connection flags in addition to command specific flags.
+
+#### tctl
+
+Use Temporal service Frontend Admin API and remove direct DB connection from the tctl admin commands.
+Implement new admin commands in github.com/temporalio/tctl repo.
+
+#### Admin API
+
+The Frontend Admin API is the functionality for tctl admin commands to work.
+
+List of commands:
+
+```bash
+$ tctl admin <command>
+```
+
```

Comment:
below is the list of existing `tctl admin` commands. Please leave a `+` or any comment to upvote the commands you would like to have first